src/App.js

```diff
@@ -3,12 +3,14 @@ import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import HomePage from './components/HomePage';
 import CategoriesPage from './components/CategoriesPage';
 import Header from './components/Header';
+import MyRedux from './MyRedux';
 
 function App() {
   return (
     <Router>
       <div className="body">
         <Header />
+        <MyRedux />
```

Comment:
- [ ] [OPTIONAL] The `MyRedux` component is not a part of the requirements , it was confusioning when I started checking your App.  Kindly keep your app clean , with needed materials 👍 

![Capture](https://github.com/ishimwepati/bookstore/assets/80458199/5f908db2-dd22-41c6-b3e8-516a30df0c1f)